core/lib/dal/migrations/20230807204005_add_fields_for_system_logs_and_compressed_state_diffs.up.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE l1_batches
+    ADD COLUMN IF NOT EXISTS system_logs BYTEA[]
+    NOT NULL DEFAULT '{}';
+
+ALTER TABLE l1_batches
+    ADD COLUMN IF NOT EXISTS compressed_state_diffs BYTEA;
```

Comment:
There is already a migration `20231018205126_l1_batches_boojum_fields.up.sql` with this columns